numbers-booleans-objects/accessing-data-in-js-objects/README.md

```diff
@@ -9,39 +9,39 @@ Every JavaScript objects has properties associated with it. And there are two wa
 
 ```js
   // Let's define a object
-  const myCar = {
-    make: 'Ford',
-    model: 'Aspire',
-    year: 2020
+  const user = {
+    name: 'Ben Stokes',
```

Comment:
Wouldn't it be better to stick with `Virat` or `Sachin` to maintain the consistency, since those are being used in other lesson?

Reply:
OK